linode/manager
# chore: Remove unused Linode components

## Description 📝

- In order to narrow the scope or RQ and other factors, I'm removing some unused code that may help lower the surface area of changes that need to be made

## How to test 🧪
- Verify these components are truly unused
- Verify automated tests pass
- Verify general functionality of Linode features in Cloud Manager
